static/src/javascripts/projects/commercial/modules/dfp/dfp-env.ts

```diff
@@ -1,14 +1,14 @@
 import config from '../../../../lib/config';
-import { getUrlVars as _getUrlVars } from '../../../../lib/url';
+import { getUrlVars } from '../../../../lib/url';
 import type { Advert } from './Advert';
 
-const getUrlVars = _getUrlVars as (arg?: string) => Record<string, string>;
-
-interface DfpEnv {
-	renderStartTime: number;
-	adSlotSelector: string;
-	hbImpl: Record<string, boolean>;
-	lazyLoadEnabled: boolean;
+interface HbImpl {
+	prebid: boolean;
+	a9: boolean;
+}
+interface IDfpEnv {
```

Comment:
What’s the benefit of keeping this interface separate from the class itself?

Reply:
Ah there doesn't actually seem to be one in this instance, will remove that as it's just repetition.